docs/FAQ.md

```diff
@@ -0,0 +1,58 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+Jujutsu's branches are akin to Git Tags, they always point to a single commit.
+To move them, use `jj branch set`. 
+
+### Where is my commit, why is it not visible in `jj log`? 
+
+If you don't find it with `jj log -r 'all()'`, it may be time to set the config
+`revsets.log` to something more matching.
+
+See [revsets], [templates] and [config] for guidance.
+
+### Is there something like `git rebase`? 
+
+For now its recommended to rebase the commit individually with 
+`jj rebase -s $changeid -d $new-parent`.
+
+In the future there may be a `histedit` command, which alleviates that.
+
+
+### How do I partially add changes from a file? 
+
+With Jujutsu you never partially add hunks, as your always amending the 
+`@` commit. You need to split it, as the `@` may already be complete.
+
+This is easy doable with `jj split your-file`
+
+
+### How can I keep my scratch files in the repository?
+
+You can keep your notes and other scratch files in the repository, if you add 
+a wildcard pattern to the `gitignore`. Something like `*.scratch` or 
```

Comment:
Nice trick!

Optional: I'd mention "either the repo's .gitignore or your global .gitignore" and link to the relevant git doc.

Reply:
All credit to @elasticdog for the trick. Done now.

Reply:
I actually go with `scratchpad.*` so that way I can rely on the extension for filetype detection within my editor, but it's a minor point.

Reply:
I've added your way as an separate paragraph.